sweep/weight_estimator.go

```diff
@@ -1,33 +1,90 @@
 package sweep
 
 import (
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcutil"
 	"github.com/lightningnetwork/lnd/input"
+	"github.com/lightningnetwork/lnd/lnwallet/chainfee"
 )
 
 // weightEstimator wraps a standard weight estimator instance.
 type weightEstimator struct {
-	estimator input.TxWeightEstimator
```

Comment:
nit: godoc could be updated to note that this estimator supports fee estimates for CPFP-related transactions

Reply:
added